🐞 Bug: New Angular Project:  Cannot find type definition file for '@public-ui/components'.

> **Note**
> - [x] Ich achte beim Erstellen des Issues darauf, dass ich **keine** fachlichen und unternehmensinternen Details preisgebe.

---

# Fehlermeldung

```ts
ERROR in error TS2688: Cannot find type definition file for '@public-ui/components'.
  The file is in the program because:
    Entry point of type library '@public-ui/components' specified in compilerOptions

webpack 5.75.0 compiled with 1 error in 4282 ms

```

## Beschreibung des Fehlers
Erstelle mit der 
## Reproduktion




Schritte zum Reproduzieren des Verhaltens:
- Erstelle neues Projekt `npm init kolibri@latest my-angular-kolibri-app`
- Wähle yarn & angular
- Starte mit npm run start

Man erhält die oben gennante Fehlermeldung angzeigt

## Erwartetes Verhalten
Es wird kein Fehler beim Angular Setup erzeugt

## Screenshots
<img width="1848" alt="image" src="https://github.com/public-ui/kolibri/assets/7667003/ede4f1a7-6db5-4421-b888-a8b041bc7992">


## Desktop
- Betriebssystem: MacOS

--- a/packages/create-kolibri/templates/angular-app/tsconfig.app.json
+++ b/packages/create-kolibri/templates/angular-app/tsconfig.app.json
@@ -15,7 +15,7 @@
 		"target": "es2015",
 		"typeRoots": ["node_modules/@types"],
 		"lib": ["es2018", "dom"],
-		"types": ["@public-ui/components", "mocha", "node"],
+		"types": ["mocha", "node"],
 		"forceConsistentCasingInFileNames": true,
 		"removeComments": false,
 		"strict": false

